src/test/kotlin/uk/gov/dluhc/printapi/rest/GetCertificateSummaryByApplicationIdIntegrationTest.kt

```diff
@@ -70,19 +83,67 @@ internal class GetCertificateSummaryByApplicationIdIntegrationTest : Integration
 
     @Test
     fun `should return certificate summary given existing application and user with valid bearer token belonging to same ero and valid group`() {
```

Comment:
Is it worth adding a test for the `404` Not Found flow?

Reply:
Definitely. I left it to be implemented as part of [EIP1-2593](https://technologyprogramme.atlassian.net/browse/EIP1-2593) subtask.

Reply:
I started working on it but saw it is dependent on the implementation of the happy path scenario, so paused and did the happy path first. I will have the PR for the not found scenario in a couple of minutes as well. 
Thanks for the approval.